go-get/README.md

```diff
@@ -25,6 +25,29 @@ The script does not require Go in the host system.
 }
 ```
 
+  Example manifest (yaml):
+```yaml
+app-id: writeas-cli
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: echo "Done"
+modules:
+  - name: writeas
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/golang/bin
+      env:
+        GOBIN: /app/bin
+        GO111MODULE: off
+      build-args:
+        - --share=network
```

Comment:
build are sandboxed, so `build-args` will be rejected. (fatal error)

Reply:
Yep: this is not the final manifest used for Flathub, but one used just to generate the source list. See the full readme for go-get here:
https://github.com/flatpak/flatpak-builder-tools/blob/master/go-get/README.md